### PR TITLE
Rename all foreign key columns in Postgres to use camel case

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
There is currently a pain point in the api where fk columns are pascal cased however the code written is in a camel case format. This means that whenever we want to interact with the database we need to run the passed parameters through a toDB/fromDB function to convert the format. 
This PR aims to rename all fk columns to camel case to allow us to remove this confusion.

### How to review 
All api tests pass and the api/front-end interact as normal.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,4253 @@
+[999D[K[999D[K[0m[7m[1m[31m FAIL [39m[22m[27m[0m [2mschema/[22m[1mresolvers.test.js[22m
```
Comment (tgandrews) [inline test.txt]: I presume this has been unintentionally committed?

Comment (SamGodwin2) [reply test.txt]: Whoops!

**Comment by SamGodwin2** [comment]
[Investigate renaming of columns in Postgres](https://trello.com/c/D0IDEKxF/587-investigate-renaming-of-columns-in-postgres)

File: migrations/20180903142820_renaming_fk_columns_in_db.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  promiseArray.push(
+    knex.schema.table("Metadata", t => {
+      t.renameColumn("QuestionnaireId", "questionnaireId");
+    })
+  );
+
+  return Promise.all(promiseArray);
+};
+
+exports.down = function(knex) {
```
Comment (tgandrews) [inline migrations/20180903142820_renaming_fk_columns_in_db.js]: Do we need this?

File: migrations/20180907154321_recreate_views_with_new_naming_conventions.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+exports.up = async function(knex, Promise) {
+  await knex.raw(`DROP VIEW "PagesView"`);
```
Comment (tgandrews) [inline migrations/20180907154321_recreate_views_with_new_naming_conventions.js]: Do we need to drop the view given we are using `CREATE OR REPLACE VIEW`?

Comment (SamGodwin2) [reply migrations/20180907154321_recreate_views_with_new_naming_conventions.js]: That's what I thought but for some weird reason, if I try to use `create or replace` without dropping the views first I get an error in the migration preventing it from happening. It just complains saying I can't rename the columns...

File: migrations/20180903142820_renaming_fk_columns_in_db.js (hunk 2 above)
Comment (SamGodwin2) [reply migrations/20180903142820_renaming_fk_columns_in_db.js]: Probably not, pretty much just added it for the sake of consistency, worth having that tech session soon so we can decide whether to be rid of them for good.

File: migrations/20180907154321_recreate_views_with_new_naming_conventions.js (hunk 3 above)
Comment (tgandrews) [reply migrations/20180907154321_recreate_views_with_new_naming_conventions.js]: That's daft 😞

**Pull request merged**